Software/Scratch/GoPiGoScratch.py

```diff
@@ -373,25 +373,25 @@ def run(self):
 				import ir_receiver
 				en_ir_sensor=1
 			try:
-				a= ir_receiver.nextcode()  # press a button on the remote 
-				if len(a) !=0: 
+				a= ir_receiver.nextcode()  # press a button on the remote
+				if len(a) !=0:
 					print a
 				else:
-					a=[0]	#eturn 0 if no keypress found 
+					a=[0]	#eturn 0 if no keypress found
 			except:
 				if en_debug:
 					e = sys.exc_info()[1]
 					print "Error reading IR sensor: " + str(a)
 			if en_debug:
 				print "IR Reading: " + str(a)
-			if en_gpg: 
+			if en_gpg:
 				s.sensorupdate({'ir':a})
-				
+
 		# Get the value from the Dexter Industries line sensor
 		elif msg.lower()=="LINE".lower():
 			try:
 				import sys
-				sys.path.insert(0, '/home/pi/Dexter/GoPiGo/Software/Python/line_follower')
+				sys.path.insert(0, '/home/pi/Dexter/DI_Sensors/Python/di_sensors/red_line_follower/line_follower')
```

Comment:
again, feel free to load it as a proper package.

Reply:
Right.